spotirec.py

```diff
@@ -189,7 +192,7 @@ def print_choices(data=None, prompt=True, sort=False) -> str:
                             f'{data[x + 2] if len(data[x + 2]) < 40 else f"{data[x + 2][0:37]}.. "}\n'
         except IndexError:
             continue
-    print(line.strip('\n'))
+    logging.info(line.strip('\n'))
```

Comment:
This print provides the user with their available choices when using custom recommendation schemes, so they should always be shown.

Reply:
Fixed in d3df488